api/pkg/apis/projectcalico/v3/felixconfig.go

```diff
@@ -442,6 +442,10 @@ type FelixConfigurationSpec struct {
 	// BPFPolicyDebugEnabled when true, Felix records detailed information
 	// about the BPF policy programs, which can be examined with the calico-bpf command-line tool.
 	BPFPolicyDebugEnabled *bool `json:"bpfPolicyDebugEnabled,omitempty"`
+	// BPFForceTrackPacketsFromIfaces forces BPF to track packets from a list of specified interfaces.
+	// Essentially this excludes the specified list of interfaces from being NOTRACKed in IP Tables.
+	BPFForceTrackPacketsFromIfaces *[]string `json:"bpfForceTrackPacketsFromIfaces,omitempty"`
```

Comment:
Think this field needs validation in libcalico-go to make sure that it is a good list of interface patterns.  That validation probably needs to be duplicated in felix config parsing.  I think we already have a regex for validating interface patterns.

Reply:
Implemented as recommended; found ifaceFilterRegex which is preferred over interfaceRegex as this accommodates wildcards e.g. "docker+"